src/protocols/secure_channel/CASESession.cpp

```diff
@@ -55,7 +55,8 @@ constexpr uint8_t kIVSR2[] = { 0x4e, 0x43, 0x41, 0x53, 0x45, 0x5f, 0x53, 0x69, 0
 constexpr uint8_t kIVSR3[] = { 0x4e, 0x43, 0x41, 0x53, 0x45, 0x5f, 0x53, 0x69, 0x67, 0x6d, 0x61, 0x52, 0x33 };
 constexpr size_t kIVLength = sizeof(kIVSR2);
 
-constexpr size_t kTAGSize = 16;
+constexpr size_t kTAGSize               = 16;
+constexpr uint32_t kMaxCHIPOpCertLength = 1024;
```

Comment:
Please document where this number is coming from.  I don't recall anything in the spec talking about certs this large...